docs/source/apps/sub_apps.md

```diff
@@ -0,0 +1,17 @@
+# Primary and dependent applications
+
+MarketPlace supports the registration of one or more applications contained within another platform, such as JupyterHub-based [AiiDalab](https://aiidalab.readthedocs.io/) and [SimPhony remote](https://simphony-remote.readthedocs.io/).
+This requires the registration of one main application for the platform (primary), and one for each of the applications contained (dependent).
+
+- Primary application:
+
+  - Represents a platform containing dependent applications.
+  - Is in charge of the authentication, as it is shared among all the dependent apps.
+    This means that when the platform is launched, the client ID and secret used are the ones returned by the MarketPlace for this application.
+  - Has a limited amount of endpoints, mainly front page/system ones.
+
+- Dependent application:
+
+  - Represents an application in the platform of the corresponding primary application.
+  - Does not need any authentication information, since it is handled centrally by the platform (and thus is common to all dependent applications).
```

Comment:
What is the meaning by "authentication information"? What do I do differently as an application provider when I register a dependent application?

Reply:
To be honest I did not go into too much detail because I expect that we will implement the changes from [this (private) issue](https://gitlab.cc-asp.fraunhofer.de/MarketPlace/platform/issue-reporting/-/issues/111) soon, and any explanation I give now about ignoring client ID and secret for the dependent app and using the primary in the platform will have to be revisited.

Reply:
@yoavnash can I merge, or should I add more info?

Reply:
Alright, let's merge. Hopefully we could address the private issue soon.